CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
+
+* Drop support for Ruby 3.1, keep supporting Ruby 3.2, use Ruby 3.3 as default https://github.com/alphagov/rails_translation_manager/pull/78 
```

Comment:
I think it would be better to have these changes in a separate commit.

Reply:
Fixed